compiler/codegen.js

```diff
@@ -56,6 +56,101 @@ const funcRef = func => {
     [ Opcodes.const, 'funcref', func.name ]
   ];
 
+  if (Prefs.indirectCallMode === 'array' && !func.internal) {
```

Comment:
For now could you just add the `padding` mode to try and simplify internals for now? ty

Reply:
Just remove `array` right? Not switch `padding` to the default?

Reply:
Yeah please